apps/web/src/routes/(marketing)/(auth)/login/+page.svelte

```diff
@@ -13,7 +13,7 @@
 	export let form: ActionData;
 
 	function clearForm() {
-		form.error = null;
+		(form.error as any) = null;
```

Comment:
Is any needed here? Usually you should always avoid using the `any` type.

Reply:
I removed it now but the reason is  b/c i was getting type error, form.error not assignable to null ,